src/coffee/categories.coffee

```diff
@@ -15,8 +16,11 @@ class Categories
     @id2fqName = {}
     @duplicateNames = []
 
-  getAll: (client) ->
-    client.categories.all().fetch()
+  getAll: (client, projectKey) ->
```

Comment:
Are you sure you will get all items here?

`client.categories.all().fetch()` from old SDK was aggregating page by page all items for you and was returning only if all items matching the query has been fetched. Unfortunately I did not find equivalent convenience function in nodejs SDK. You could use `client.process` here but then you would need to aggregate all paged results in your callback on your own.

Reply:
@butenkor Thanks. Modified here ff7eb1573ccd88959978a381fc5268af7e5cf9e5